all.json

```diff
@@ -1,6 +1,25 @@
 {
 	"allow": [],
 	"deny": [
+		"test1.consultando.eu",
+		"terra-supports.com",
+		"connect-wallet-recovery.annelouisekrogh.dk",
+		"mainsync.net",
+		"bit-vestnetwork.com",
+		"fixingprotocol.com",
+		"securetokenrestore.com",
+		"easywebsyncrination.com",
+		"smartdappdata.io",
+		"smartcointechsolution.art",
+		"smartdappsauthorizatoin.live",
+		"smartcointechsolutiion.xyz",
+		"smartcointechsolution.live",
+		"itsmy-asset.com",
+		"configdevminter.online",
+		"mainbscrectify.com",
+		"micropool-defi.co",
+		"drasper27.github.io",
```

Comment:
Taken down by GH.

Based on history, seems quite old - https://github.com/drasper27

Reply:
Still live when opened in different browser. Better to block it anyway, no?

Reply:
Ah I get it. The content is still live though. Check out the urlscan + screenshots. It's still fresh. @jacogr

Reply:
Weird, went to the URL directly, it is down. No issues, doesn't hurt having it.

What is perplexing is that it stayed up so long, the repos on his account `live1`/etc is quite old. He/she probably just kept it in the bag and surfaced it recently.

Reply:
Yeah, it just recently circulating on several discord servers. Really risky to let it be in the wild. Thanks for reviewing!